shell/platform/darwin/ios/framework/Source/accessibility_bridge_test.mm

```diff
@@ -394,9 +393,9 @@ - (void)testAnnouncesRouteChangesWhenNoNamesRoute {
 
   // Notification should focus first focusable node, which is node1.
   XCTAssertEqual([accessibility_notifications count], 1ul);
-  SemanticsObject* focusObject = accessibility_notifications[0][@"argument"];
-  XCTAssertEqual([focusObject uid], 2);
-  XCTAssertEqualObjects([focusObject accessibilityLabel], @"node2");
```

Comment:
This seems wrong in retrospect, right?  Node2 wasn't a route but was getting the route change notificaiton call?